stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -148,8 +148,21 @@ extension UnsafeRawBufferPointer.Iterator: IteratorProtocol, Sequence {
   public mutating func next() -> UInt8? {
     if _position == _end { return nil }
```

Comment:
This _should_ compile to a single comparison, but doesn't. I'm opening an LLVM bug to see what we can do about that.